src/en/space-station-14/departments/security/proposals/security-improvements.md

```diff
@@ -0,0 +1,120 @@
+
+I suggest that holographic projectors get a single useful trait - barriers created by the projector won't have collision for anyone with Brig access. Others will still be able to vault, just as before. 
+
+This turns the beforementioned situation with a prisoner in a following interaction:
+- Warden brings a prisoner into a cell;
+- Warden places a barrier right outside of a cell;
+- Warden uncuffs the prisoner;
+- Warden walks out of the cell without any troubles;
+- Prisoner might try to escape by vaulting the barrier, but will probably get disabled by a Warden.
+
+This interaction loop leaves counterplay, but makes security job easier.
+
+## Security Records
+
+Security Records have improved a lot, but there is still 1 function (except from QoL such as filtering by status) that is missing.
+
+Currently, security officers either have to update entries while being in security or to rely on warden/HoS entirely for that (since not every map has records mapped in general security area, some have the computer only in warden room).
+
+The solution for this is PDA security records cartridge - probably not a roundstart option, but rather a T1/T2 research. With this, officers will be able to set statuses manually.
+
+## Alert Codes Rework
+
+Currently, alert codes are in a weird position. Some of them are linear in order of ascending severity: Green is "All Clear", Blue is "Be Alert" and Red is "Everything Is Falling Apart". However, Violet, Yellow and Admeme-type alerts are different - these are meant to signal *what* kind of an emergency is ongoing, not *how severe* it is. Before SOP was removed in rules refactor, it also affected the order of command authority, often in confusing ways.
+
+This is an example of real-life codes for emergencies at hospitals. Each emergency type is represented by a color.
+
+![types of emergencies graded by colors](https://i.imgur.com/FnhDYyG.png)
+
+
+For SS14, I suggest to adapt this system. Here's an outline of how it would look like:
+
+![new system proposal](https://i.imgur.com/IliqzBu.png)
+
+You will also be able to set multiple alerts.
+
+For the sake of simplicity, I propose that on LRP, any emergency allows for searches or using high-grade security gear. On MRP, however, security would have to follow this table: if there are no emergencies allowing for random searches, security cannot do them.
+
+This change is actually pretty minor: command already announces what is going on when changing alerts currently. This change will allow for more context for latejoiners and for crew that didn't get the context of said alert level raise for some reason.
+
+## Alarm Consequences
+
+For the sake of modularity, this will be discussed separately from the Alert Codes rework, using the current alert system. 
+
+Currently, there are zero consequences for abusing alarm system. You can just set the alarm to blue upon any minor crime being committed in order to give security a random search pass.
```

Comment:
wont this just make the problem worse

1. hos stubs toe on the door entering the bridge
2. code blue for the next hour
3. sec gets free guns as well as free search pass

Reply:
That's valid, I should probably add something else to blue alert